usage_test.go

```diff
@@ -24,6 +26,46 @@ func TestFormatTwoColumns(t *testing.T) {
 	assert.Equal(t, expected, buf.String())
 }
 
+func TestOptionalSubcommandsUsage(t *testing.T) {
+	if os.Getenv("BE_CRASHER") == "1" {
```

Comment:
It should not be necessary to do this.

See other tests in this file for how to capture output (`.Writers(...)`), and trap termination (`.Terminate(...)`).